hudi-common/src/main/java/org/apache/hudi/common/util/CollectionUtils.java

```diff
@@ -243,40 +243,31 @@ public static boolean elementsEqual(Iterator<?> iterator1, Iterator<?> iterator2
     return !iterator2.hasNext();
   }
 
-  @SafeVarargs
-  public static <T> Set<T> createSet(final T... elements) {
-    return Stream.of(elements).collect(Collectors.toSet());
-  }
-
-  public static <K,V> Map<K, V> createImmutableMap(final K key, final V value) {
-    return Collections.unmodifiableMap(Collections.singletonMap(key, value));
-  }
```

Comment:
removed. use singletonMap() instead. this overloading interferes with `createImmutableMap(final Pair<K, V>... elements` when there are 2 `Pair`s